packages/jellyfish-api-core/src/category/governance.ts

```diff
@@ -70,7 +70,7 @@ export class Governance {
    * @return {Promise<ProposalInfo>} Information about the proposal
    */
   async getProposal (proposalId: string): Promise<ProposalInfo> {
-    return await this.client.call('getproposal', [proposalId], 'number')
+    return await this.client.call('getproposal', [proposalId], { amount: 'bignumber' })
```

Comment:
Learn a new thing today. I dunno this can be written like this. 
So, only amount is returned as `bignumber`. The rest is still `number`.